central/networkgraph/entity/datastore/datastore_impl.go

```diff
@@ -446,6 +446,16 @@ func (ds *dataStoreImpl) DeleteExternalNetworkEntitiesForCluster(ctx context.Con
 	return nil
 }
 
+func (ds *dataStoreImpl) RemoveOrphanedEntities(ctx context.Context) (int64, error) {
+	if ok, err := ds.writeAllowed(ctx, "__glob"); err != nil {
+		return 0, err
+	} else if !ok {
+		return 0, sac.ErrResourceAccessDenied
+	}
```

Comment:
@rhybrillou I am trying to set appropriately the required authorization to run `RemoveOrphanedEntities()`, which should require to be a global admin. The only context calling it is [the one of the main pruner](https://github.com/stackrox/stackrox/blob/3657ac26df3d557ad2c4adf92619cb92574d2269/central/pruning/pruning.go#L73) (In theory, SAC is not required here, but still trying for the sake of completeness). `__glob` works, but I am sure there is something better to use here.

Reply:
As discussed, I implemented SAC by introducing a new resource dedicated to pruning

https://github.com/stackrox/stackrox/pull/13117/commits/63f5c8e9b38a8639749adce0fe602e0f2f6d8c63